app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,15 @@
+# controller for activation
+class AccountActivationsController < ApplicationController
+  def edit
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for edit is too high. [15.59/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)

<!-- Comment by [SideCI](https://sideci.com) -->